### ISSUE TITLE
AT&T Unix PC floppy images

### ISSUE THREAD

**Issue by cb88** [davidgiven/fluxengine#279]
They just use a standard 10 or 8 sector 5.25in IBM floppy with a cpio image on the disk usually instead of a fat filesystem.

I see other people are having some success with reading similar disk formats so I'll have to give this a try and upload a a few images for you to inspect here once I've assembled a fluxengine. Would it be possible to detect the number of sectors if you know the machine type ahead of time?

I'd really like to have a tool to image disks and get data onto and off of my system that way (probably link it up via serial later after I get some software onto it).

**Comment by davidgiven** [comment]
Yes, the IBM reader (`fluxengine read ibm`) will autodetect nearly everything and figure out the format. The one thing it can't know is whether the start sector ID is 0 or 1 --- you'll need to set this via `--decoder.ibm.sector_id_base`. Once determined, adding a profile for the Unix PC is easy; see `src/readables/acornadfs.textpb`. (In fact, if the start sector ID is 0 you can probably just _use_ the acornadfs profile.) Let me know if it works; adding support for this would be great!
